src/components/Review/Comment-info.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+import {getCommentById} from '../Utils/Api'
+
+export const CommentInfo = ({reviewId}) => {
+    const [comments, setComments] = useState([])
+
+    useEffect(() => {
+        getCommentById(reviewId)
+          .then((response) => {
+            console.log(response);
+            setComments(response);
+          })
+      }, []);
+
+    return (
+        <ul className="Cbox" >
+            {comments.map((comment)=> {
+               return <li key={comment.comment_id} className='comment' >
+                    <header>{comment.author}</header>
+                    <p>{comment.votes}</p>
```

Comment:
Consider changing the paragraphs to have headers and then paragraphs to distinguish titles from comments etc...